### ISSUE TITLE
Running pipelines via the UI hangs and 500's are encountered.

### ISSUE THREAD

**Issue by pandemicsyn** [meltano/meltano#6210]
Original user report: https://meltano.slack.com/archives/C01TCRBBJD7/p1655302068277449?thread_ts=1655235602.029959&cid=C01TCRBBJD7

```
The pipeline shows in the UI for me but when I try to run the pipeline, it's stuck and I get the below error
2022-06-15T14:04:31.879525Z [error    ] Exception on /api/v1/orchestrations/jobs/state [POST] 
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/site-packages/flask_restful/__init__.py", line 271, in error_router
    return original_handler(e)
  File "/usr/local/lib/python3.8/site-packages/flask_restful/__init__.py", line 271, in error_router
    return original_handler(e)
  File "/usr/local/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/local/lib/python3.8/site-packages/meltano/api/controllers/orchestrations.py", line 207, in job_state
    state_ids = poll_payload["state_ids"]
KeyError: 'state_ids'
2022-06-15T14:04:31.879843Z [info     ] Error: 500 Internal Server Error: The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.
```

This exception and `state_ids` being missing isn't the direct error, its a side effect, for some reason the client isn't passing along a valid POST object as its missing `state_ids` (side note this should probably return a Bad Request or Pre-con failed, not a 5xx).  We'll need to double-check and see why `state_ids` is actually not being included.

## Impact

This issue blocks users from running pipelines in the Meltano UI web GUI.

## Workaround

As a workaround, users would need to run EL workloads using the CLI (`meltano schedule run`) or in Airflow.

**Comment by pandemicsyn** [comment]
Quick addition (and confirmation) I think

The`/v1/orchestrations/run` returns a stateId as expected:

```
{"stateId":"gitlab-to-jsonl2"}
```

But the post to `/v1/orchestrations/jobs/state` is submiting a malformed body trying to use the old `jobId` key:

![Screen Shot 2022-06-15 at 11 39 55 AM](https://user-images.githubusercontent.com/333354/173880553-5688762a-24ad-44a0-998f-b07e8675f89c.png)

Which I think is just because when we refactored https://github.com/meltano/meltano/pull/6009 we forgot to also make the equivalent changes in the UI (ala https://github.com/meltano/meltano/blob/dfd9650170ff97fd69fdff7ac8dc7dc0841bf663/src/webapp/src/components/pipelines/Pipeline.vue)

**Comment by edgarrmondragon** [comment]
>  we forgot to also make the equivalent changes in the UI

@pandemicsyn ah yes 🤦‍♂️

**Comment by aaronsteers** [comment]
@cjohnhanson - Do you think you could pick this up tomorrow as hi pri / high urgency?

**Comment by cjohnhanson** [comment]
@aaronsteers yeah, I can do that.

**Comment by alexmarple** [comment]
@pandemicsyn @edgarrmondragon hmm. so is this just a simple swapping of jobId with stateId? i'll dig deeper in a few but if that's all that needs doing i can do that today.

this ties into #3378 and it's for the best i put it on my to do list this week. would this be best served with some pair programming tomorrow @cjohnhanson ?

**Comment by alexmarple** [comment]
@edgarrmondragon i'm always so suspicious when it's that easy. ok i'll make that correction and test it.

**Comment by aaronsteers** [comment]
Thanks, @alexmarple and @cjohnhanson . This'll be worth a patch release, I think. Would be great to have it out by Friday, if not tomorrow.

**Comment by cjohnhanson** [comment]
@alexmarple thanks for jumping on this, I'm afk right now but can hop back on later if needed, otherwise will plan on tackling first thing tomorrow if you don't already have a PR by then. @aaronsteers I'm very confident we can get this released tomorrow, since it's just a find+replace

**Comment by alexmarple** [comment]
https://github.com/meltano/meltano/pull/6216

Here's a PR but the issue persists after swapping out instances of jobId with stateId. Will continue to investigate in the morning.

**Comment by pandemicsyn** [comment]
@alexmarple dropped a note on that PR but fwiw what you've currently got in the PR resolved the issue for me. Was able to run the pipeline and view the log.

**Comment by alexmarple** [comment]
Whoops I forgot to put a close comment in my PR. This was closed by #6216. 

I think this has been discussed before but given that the pipeline just hangs what do you all think about adding a "stop pipeline" button? @pandemicsyn @cjohnhanson @aaronsteers @edgarrmondragon

**Comment by edgarrmondragon** [comment]
> Whoops I forgot to put a close comment in my PR. This was closed by #6216.
> 
> I think this has been discussed before but given that the pipeline just hangs what do you all think about adding a "stop pipeline" button? @pandemicsyn @cjohnhanson @aaronsteers @edgarrmondragon

@alexmarple you mean here

<img width="547" alt="Screen Shot 2022-06-16 at 14 28 38" src="https://user-images.githubusercontent.com/16805946/174149724-bb024a46-5564-404c-9cff-d1ed59515b79.png">?
